src/main/resources/org/archive/commons.properties

```diff
@@ -0,0 +1,5 @@
+operator=<enter operator name here>
+publisher=<enter publisher name here>
+wat.warcinfo.description=<enter warc info description here>
```

Comment:
For users who don't change the commons.properties file, won't they have the literal strings of  `<enter operator name here>` , `<enter publisher name here>`, and `<enter warc info description here>` in their WAT files? If there is no documentation for using WAT extractor (please point me to the documentation if it exists), people may not change the commons.properties file, so the default values here could be an issue. Perhaps others could voice opinions on this.